src/components/Header/Header.module.css

```diff
@@ -0,0 +1,99 @@
+.header {
+    width: 100%;
+    height: 80px;
+    z-index: 9990;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 20px;
+    box-sizing: border-box;
+    position: fixed;
+
+}
+
+.logoDiv {
+    flex: 0 0 auto;
+
+}
+
+.airbnbHomeButton {
+    cursor: pointer;
+    height: 80px;
+    width: 80px;
+
+}
+
+.logoAirbnb {
+    height: 80px;
+    width: 80px;
+}
+
+.toggleButtonsStaysExperiences {
```

Comment:
According to Airbnb header the  button that is active has bold font face and the not active button has regular font face and some transparency ( looks grey instead of black) I dont see that styling in youer component
<img width="1533" alt="Screenshot 2024-09-30 at 15 33 43" src="https://github.com/user-attachments/assets/eb744345-004c-48ac-b110-feaf6b530a8a">

Reply:
done